src/app/colorpicker/colorpicker.demo.html

```diff
@@ -3,7 +3,7 @@
     <form>
       <div class="field">
         <label for="bgcolor" class="label">Background Color</label>
-        <input soho-colorpicker name="bgcolor" [(ngModel)]="model.color" (change)="onChange($event)" />
+        <input soho-colorpicker name="bgcolor" [editable]="true" [disabled]="false" [(ngModel)]="model.color" (change)="onChange($event)" />

```

Comment:
Could we extend the example to show the value changing dynamically?

Reply:
@bthharper  sure sure.